app/views/how-tos/index.html

```diff
@@ -14,13 +14,33 @@
 {% block content %}
   <div class="nhsuk-grid-row">
 
-    <div class="nhsuk-grid-column-two-thirds">
+    <div class="nhsuk-grid-column-full">
 
       <h1>Guides</h1>
 
       <p class="nhsuk-lede-text">These guides will show you how to use the prototype kit, from creating a simple page to building complex user journeys.</p>
 
-      <h2>Build a basic prototype</h2>
+    </div>
+
+
+
+    <div class="nhsuk-grid-column-one-half">
+      <h2 class="nhsuk-heading-s">Download and install</h2>
+
+      <ul class="nhsuk-list">
+        <li>
+          <a href="/install/index">Get started with the NHS prototype kit</a>
```

Comment:
Don’t think we need to use “NHS prototype kit” here as it’s implied from the context of the whole site being about it?

```suggestion
          <a href="/install/index">Install guide</a>
```